js/main.js

```diff
@@ -69,32 +68,38 @@ function displayEntries(entry) {
 }
 
 function loadEntries() {
-  $entryViewList.innerHTML = '';
+  if (data.entries.length === 0) {
+    var $noEntries = document.createElement('p');
+    $noEntries.textContent = 'No Entries have been recorded.';
+    $noEntries.setAttribute('class', 'no-entries');
+    $entryViewList.appendChild($noEntries);
+  }
   for (var i = 0; i < data.entries.length; i++) {
     var $entry = displayEntries(data.entries[i]);
     $entryViewList.appendChild($entry);
   }
+  if (data.view === 'entry-form') {
+    showEntryForm();
+  } else {
+    showEntriesList();
+  }
 }
 
-function entriesPage() {
+function showEntriesList() {
   // go to entries
   $viewEntry.setAttribute('class', 'view');
   $createEntry.setAttribute('class', 'view hidden');
-  loadEntries();
   data.view = 'entries';
 }
 
-function createEntryPage() {
+function showEntryForm() {
   // go to create entry
   $viewEntry.setAttribute('class', 'view hidden');
   $createEntry.setAttribute('class', 'view');
   data.view = 'entry-form';
 }
 
-function onLoadSite() {
-  if (data.view === 'entry-form') {
-    createEntryPage();
-  } else {
-    entriesPage();
-  }
+function renderEntry() {
```

Comment:
I meant to rename `displayEntries` to `renderEntry`, since it is only building one entry's DOM tree, so the plural is inaccurate, and `display` isn't accurate either, since you are creating something, not just making it visible.